ledger/src/bank_forks_utils.rs

```diff
@@ -149,10 +149,6 @@ fn load_from_snapshot(
         deserialized_bank.set_shrink_paths(shrink_paths);
     }
 
-    if process_options.accounts_db_test_hash_calculation {
```

Comment:
this is no longer necessary here since we now use the non-index scan and support validation of index and non-index scans.